draft-ietf-tls-tls13.md

```diff
@@ -566,6 +566,14 @@ compatible with previous versions, all versions of TLS incorporate a
 versioning mechanism which allows clients and servers to interoperably
 negotiate a common version if one is supported.
 
+This document supersedes and obsoletes previous versions of TLS including
+version 1.2 {{RFC5246}}.  It also obsoletes the TLS ticket mechanism defined in
```

Comment:
nit: please wrap new text to around 80 char (we're in no way consistent, but this paragraph has particularly long lines and doesn't fit in GitHub's columns)